Code/Engine/GameEngine/GameApplication/Implementation/GameApplicationInit.cpp

```diff
@@ -223,6 +224,11 @@ void ezGameApplication::Init_SetupDefaultResources()
   {
     ezResourceManager::AllowResourceTypeAcquireDuringUpdateContent<ezDecalAtlasResource, ezTexture2DResource>();
   }
+
+  // Font
```

Comment:
Have a look at the startup system:
http://ezengine.net/runtime/configuration/startup.html

That allows you to set such stuff up in your code, so you don't need to modify the application code.